R/lintr-deprecated.R

```diff
@@ -0,0 +1,186 @@
+#' Deprecated functions in lintr
+#'
+#' These functions have either been deprecated from lintr.
+#'
+#' @keywords internal
+#' @noRd
+NULL
+
+#' Closed curly linter
+#'
+#' Check that closed curly braces are on their own line unless they follow
+#' an else, a comma, or a closing bracket.
+#'
+#' @param allow_single_line if `TRUE`, allow an open and closed curly pair on the same line.
+#' @evalRd rd_tags("closed_curly_linter")
+#' @seealso
+#'   [linters] for a complete list of linters available in lintr. \cr
+#'   <https://style.tidyverse.org/syntax.html#indenting>
+#' @keywords internal
+#' @export
+closed_curly_linter <- function(allow_single_line = FALSE) {
```

Comment:
We should put them in the same .Rd as well

Reply:
Sounds good.

What about tests? Should they also be moved to a single file?

Reply:
That part's not as important, I'd leave them where they are to make the git history cleaner.

Reply:
Okay, I personally like to keep `test-*.R` files in sync with `.R` files so that if one is removed, then the test file can be removed as well. But I think this is definitely a matter of personal taste.

Retaining the tests in separate files as before.